lib/ansible/modules/storage/netapp/na_ontap_cifs_server.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/python
+""" this is cifs_server module
+
+ (c) 2018, NetApp, Inc
+ # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+
+import traceback
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_native
+import ansible.module_utils.netapp as netapp_utils
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: na_ontap_cifs_server
+short_description: cifs server configuration
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.6'
+author: chhaya gunawat (chhayag@netapp.com)
+
+description:
+    - Creating / deleting and modifying the CIF server .
+
+options:
+
+  state:
+    description:
+    - Whether the specified cifs_server should exist or not. Values: present/absent
+    required: false
+    default: present
+
+  service_state:
+    description:
+    - CIFS Server Administrative Status. Values: started/stopped
+    required: true
+
+  cifs_server_name:
+    description:
+    - Specifies the cifs_server name.
+    required: true
+
+  admin_user_name:
+    description:
+    - Specifies the LIF's home node.
+    required: false
+
+  admin_password:
+    description:
+    - Specifies the role of the LIF.
+    required: false
+
+  domain:
+    description:
+    - The Fully Qualified Domain Name of the Windows Active Directory this CIFS server belongs to.
+    required: false
+
+  workgroup:
+    description:
+    -  The NetBIOS name of the domain or workgroup this CIFS server belongs to.
+    required: false
+
+  vserver:
+    description:
+    - The name of the vserver to use.
+    required: true
+
+'''
+
+EXAMPLES = '''
+    - name: Create cifs_server
+      na_ontap_cifs_server:
+        state: present
+        vserver: svm1
+        hostname: "{{ netapp_hostname }}"
+        username: "{{ netapp_username }}"
+        password: "{{ netapp_password }}"
+
+    - name: Delete cifs_server
+      na_ontap_cifs_server:
+        state: absent
+        cifs_server_name: data2
+        vserver: svm1
+        hostname: "{{ netapp_hostname }}"
+        username: "{{ netapp_username }}"
+        password: "{{ netapp_password }}"
+
+'''
+
+RETURN = '''
+    changed: True/False
```

Comment:
This is not a properly formatted return block. Leave it empty if you're not returning anything beyond the standard values.